job-profile-wrangler/src/main/java/org/folio/RepoObject.java

```diff
@@ -0,0 +1,8 @@
+package org.folio;
+
+import org.folio.graph.edges.RegularEdge;
+import org.folio.graph.nodes.Profile;
+import org.jgrapht.Graph;
+
+public record RepoObject(int repoId, Graph<Profile, RegularEdge> graph) {
```

Comment:
@okolawole-ebsco  I think it would be better to add some documentation here. Because this is public entity and not inside the tests directory.

Reply:
added javadoc